Templates/editcourse.html

```diff
@@ -34,21 +34,34 @@ <h4>Edit Course</h4>
                 <th>Course Location</th>
                 <th>Course Hours</th>
                 <th>Course Days</th>
-                <th>Course Name</th>
                 <th>Instructor Name</th>
                 <th>TA Name</th>
                 <th>Labs</th>
             </tr>
-             {% for Courses in Course %}
+             {% for Course in Course_list %}
+                 {% if change_course.courseID  == Course.courseID %}
+                     <td><input type="text" id= name=id value={{ Course.courseID }}></td>
+                     <td><input type="text" id=name name=name value={{ Course.name }}></td>
+                     <td><input type="text" id=location name=location value={{ Course.location }}></td>
+                     <td><input type="text" id=hours name=hours value={{ Course.hours }}></td>
+                     <td><input type="text" id=days name=days value={{ Course.days }}></td>
+{#                     <td><input type="text" id=instructor name=instructor value={{ Course.instructor }}></td>#}
```

Comment:
```suggestion
                     <td>
                     <select name="instructor" id="instructor">
                         {% for UserProfile in UserProfile_list &}
                             {% if UserProfile.userType == "INSTRUCTOR"
                                 <option value ={{ UserProfile.userID }}> {{ UserProfile.username }} </option>
                             {% endif %}
                         {{% endfor %}
                     </select>
                     </td>
```
Potential idea for how to implement input for instructor. This requires passing a list of all UserProfiles called UserProfile_list to be passed to the render of this page.

Reply:
Added this in